drivers/ma_yya_report/app/models/ma_yya_report/report.rb

```diff
@@ -0,0 +1,229 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module MaYyaReport
+  class Report < SimpleReports::ReportInstance
+    include Rails.application.routes.url_helpers
+    include Reporting::Status
+
+    def run_and_save!
+      start
+      create_universe
+      report_results
+      complete
+    end
+
+    def start
+      update(started_at: Time.current)
+    end
+
+    def complete
+      update(completed_at: Time.current)
+    end
+
+    def title
+      'MA YYA Report'
+    end
+
+    def url
+      ma_yya_report_warehouse_reports_report_url(host: ENV.fetch('FQDN'), id: id, protocol: 'https')
+    end
+
+    private def create_universe
+      # FIXME Look back 1 year to check for previously reported clients
```

Comment:
this still a fix me?

Reply:
Nope!

Reply:
do you want to remove it from the code then?